### DISCUSSION TITLE
Subir um projeto maven para o mvn repository

### DISCUSSION THREAD

**Discussion by mahenrique94** [frontendbr/forum#886] [Perguntas e Respostas]
Bom dia galera, estou precisando subir o jar do meu projeto maven para a central de repositório do maven: https://mvnrepository.com/

Algum tutorial, dica, post, sobre como fazer isso?

Abraços

**Comment by felipefialho** [comment]
@mahenrique94 Nunca usei, nem conheço alguém que use. Mas estou olhando a [documentação deles](https://maven.apache.org/guides/mini/guide-central-repository-upload.html), parece ser bem completa.
